From b4e9ea9905046eaed21488f22ed0383a49f63472 Mon Sep 17 00:00:00 2001
From: Ohad Ben-Cohen <ohad@wizery.com>
Date: Wed, 14 Dec 2011 13:01:59 +0200
Subject: [PATCH] remoteproc: add Kconfig menu

Add a dedicated Kconfig menu for the remoteproc drivers, so they
don't show up in the main driver menu.

Signed-off-by: Ohad Ben-Cohen <ohad@wizery.com>
---
 drivers/remoteproc/Kconfig |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/drivers/remoteproc/Kconfig b/drivers/remoteproc/Kconfig
index 396c97a..00fb72f 100644
--- a/drivers/remoteproc/Kconfig
+++ b/drivers/remoteproc/Kconfig
@@ -1,3 +1,5 @@
+menu "Remoteproc drivers"
+
 # REMOTEPROC gets selected by whoever wants it
 config REMOTEPROC
 	tristate
@@ -22,3 +24,5 @@ config OMAP_REMOTEPROC
 
 	  It's safe to say n here if you're not interested in multimedia
 	  offloading or just want a bare minimum kernel.
+
+endmenu
-- 
1.7.0.4

